api/woeip/apps/air_quality/serializers.py

```diff
@@ -97,9 +98,10 @@ def create(self, validated_data):
         missing_file_errors = []
         for upload_file in upload_files:
             file_name = upload_file.name
-            if "dustrak" in file_name.lower():
+            _, file_ext = os.path.splitext(file_name)
+            if file_ext == '.csv':
                 dustrak_upload_file = upload_file
-            elif "gps" in file_name.lower():
+            elif file_ext == '.log':
```

Comment:
The current error message asks the user to ensure that "gps" is in the file name. Can we update the message to mention that it's now looking for a `.log` file?

Reply:
same with the dustrak error message.

Reply:
Ah thanks, good catch!